modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,188 @@
+import numpy as np
+import cv2 as cv
+import glob
```

Comment:
`glob` is no longer used

Reply:
For now I am using it to create glob of frames.

Reply:
`glob` is removed.